packages/graphback-cli/src/templates/configTemplates.ts

```diff
@@ -70,21 +62,60 @@ export const askForClient = async (): Promise<boolean> => {
  * Create config file with db info
  */
 export const createConfig = async (database: string, client: boolean) => {
-  const configPath = `${process.cwd()}/graphback.json`
+  const configPath = `${process.cwd()}/.graphqlrc.yml`
 
   const dockerComposePath = `${process.cwd()}/docker-compose.yml`
-  const config = {}
   const [dbConfig, dockerCompose] = getConfig(database)
-  config['db'] = {
-    dbConfig: JSON.parse(dbConfig),
-    database: database
+  const graphqlConfig: IGraphQLConfig = {
+    schema: './src/schema/*.gql',
```

Comment:
Why are we now preferring .gql files over .graphql files?

Reply:
That is a bug. We need to change type name to graphql instead of gql